src/js/jobs/components/JobList.tsx

```diff
@@ -35,15 +33,21 @@ const initialState = ["preparing", "running"];
  */
 export default function JobsList() {
     const [states] = useUrlSearchParamsList("state", initialState);
-    const { data, isPending, fetchNextPage, isFetchingNextPage, hasNextPage } = useInfiniteFindJobs(states);
+    const [urlPage, setUrlPage] = useUrlSearchParams<number>("page");
+    const { data, isPending } = useFindJobs(Number(urlPage) || 1, 25, states);
+
+    useEffect(() => {
+        if (data && page > page_count) {
+            setUrlPage(page_count);
+        }
+    }, [data]);
```

Comment:
What is the plausibility of instead integrating this into the pagination component? I feel like not exceeding the max page count is something we want pretty universally for our paginated components